test/test.js

```diff
@@ -225,59 +226,67 @@ function noWarnings(input) {
 
   assert.equal(frequency, missingSourceCount.length);
 }
 
 function invalidMatrixDataWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var invalidDataCount = network.warnings.filter(function(x){return x.warningCode=="INVALID_DATA"});
+  var invalidDataCount = network.warnings.filter(function(x){return x.warningCode === "INVALID_DATA"});
 
   assert.equal(frequency, invalidDataCount.length);
 }
 
 function missingTargetWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var missingTargetCount = network.warnings.filter(function(x){return x.warningCode=="MISSING_TARGET"});
+  var missingTargetCount = network.warnings.filter(function(x){return x.warningCode === "MISSING_TARGET"});
 
   assert.equal(frequency, missingTargetCount.length);
 }
 
 function randomDataWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var randomDataCount = network.warnings.filter(function(x){return x.warningCode=="RANDOM_DATA"});
+  var randomDataCount = network.warnings.filter(function(x){return x.warningCode === "RANDOM_DATA"});
 
   assert.equal(frequency, randomDataCount.length);
 }
 
 function emptyRowWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var emptyRowCount = network.warnings.filter(function(x){return x.warningCode=="EMPTY_ROW"});
+  var emptyRowCount = network.warnings.filter(function(x){return x.warningCode === "EMPTY_ROW"});
 
   assert.equal(frequency, emptyRowCount.length);
 }
 
 function invalidNetworkSizeWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var invalidNetworkSizeCount = network.warnings.filter(function(x){return x.warningCode=="INVALID_NETWORK_SIZE"});
+  var invalidNetworkSizeCount = network.warnings.filter(function(x){return x.warningCode === "INVALID_NETWORK_SIZE"});
 
   assert.equal(frequency, invalidNetworkSizeCount.length);
 }
 
 function extraneousDataWarning(input, frequency) {
   var sheet = xlsx.parse(input),
     network = spreadsheetController.parseSheet(sheet);
-  var extraneousDataWarning = network.warnings.filter(function(x){return x.warningCode=="EXTRANEOUS_DATA"});
+  var extraneousDataWarning = network.warnings.filter(function(x){return x.warningCode === "EXTRANEOUS_DATA"});
 
   assert.equal(frequency, extraneousDataWarning.length);
 }
 
+function incorrectlyNamedSheetWarning(input, frequency){
+  var sheet = xlsx.parse(input),
+    network = spreadsheetController.parseSheet(sheet);
+  var incorrectlyNamedSheetWarning = network.warnings.filter(function(x){return x.warningCode === "INCORRECTLY_NAMED_SHEET"});
+
+  assert.equal(frequency, incorrectlyNamedSheetWarning.length);
+}
+
```

Comment:
There might be other places where we can separate variable declarations into separate `var` statements, but we are not obligated to revise those for now. We want to revise _these_ because your work happens to encounter them. These little improvements, if done consistently, will allow the code to evolve toward a cleaner, more consistent, and less error-prone style.